Makefile

```diff
@@ -17,14 +34,3 @@ release-zip: all
 	sed -i -e "s/__VERSION__/$(EXTVERSION)/g"  ./pg_anonymize-$(EXTVERSION)/META.json
 	zip -r ./pg_anonymize-$(EXTVERSION).zip ./pg_anonymize-$(EXTVERSION)/
 	rm ./pg_anonymize-$(EXTVERSION) -rf
-
-
-DATA = $(wildcard *--*.sql)
```

Comment:
You're right, this one is actually unnecessary.  I copy/pasted the makefile from another project and didn't clean it enough.  It will be easy enough to reintroduce if this project ever ships an extension on top of the module.

Reply:
Thank you. I took the liberty of splitting the commit into smaller chunks. This way, it might be easier to merge the bits you see as useful.